### ISSUE TITLE
Request: Method for lookups based on pre-computed hashes

### ISSUE THREAD

**Issue by glmcdona** [yankun1992/fastbloom#2]
I have a use-case where I need to look up a key within multiple bloom filters of identical fixed size and properties. Unioning the filters together doesn't help in this scenario because I need to identify which bloom filters contain the key. Ideally, I'd create the hash of the key once, then look up the set of hashes within the multiple bloom filters to avoid the re-hashing of the content.

From python, maybe it'd look like:

```python
hashes = bloom.get_hashes('hello')
if bloom.contains_hashes(hashes):
   ... do some stuff ...
```

**Comment by yankun1992** [comment]
Hi, I have released a new version (0.4.0) to support this feature, you can find example at [python unittest code](https://github.com/yankun1992/fastbloom/blob/main/py_tests/test_bloom.py#L106). Can this version work for you?

**Comment by glmcdona** [comment]
Wow, super quick turnaround! Looks perfect. Giving it a try now.
